src/System.Runtime.Extensions/tests/System/IO/PathTests.cs

```diff
@@ -36,7 +36,6 @@ public static void ChangeExtension(string path, string newExtension, string expe
 
         [Theory]
         [InlineData(null, null)]
-        [InlineData("", null)]
```

Comment:
This was already fixed here:
https://github.com/dotnet/corefx/pull/19440/commits/e1755b5a62cd07037d5488d459a61cf97378226d
(had to be in order to merge in the coreclr update)